integrationtests/tests/fixtures/mod.rs

```diff
@@ -212,15 +229,23 @@ pub async fn fixtures(num_peers: u16) -> anyhow::Result<Fixtures> {
                 };
             let bitcoin = RealBitcoinTest::new(&url, bitcoin_rpc.clone());
 
-            let lightning = RealLightningTest::new().await;
-
-            let lnrpc_addr = env::var("FM_GATEWAY_LIGHTNING_ADDR")
-                .expect("FM_GATEWAY_LIGHTNING_ADDR not set")
-                .parse::<Url>()
-                .expect("Invalid FM_GATEWAY_LIGHTNING_ADDR");
-            let lnrpc: DynLnRpcClient = NetworkLnRpcClient::new(lnrpc_addr).await.unwrap().into();
-            let lnrpc_adapter = LnRpcAdapter::new(lnrpc);
+            // lightning
+            let socket_cln = PathBuf::from(dir.clone()).join("cln/regtest/lightning-rpc");
+            let rpc_cln = Arc::new(Mutex::new(ClnRpc::new(socket_cln).await.unwrap()));
+            let lnd_rpc_addr = env::var("FM_LND_RPC_ADDR").unwrap();
+            let lnd_macaroon = env::var("FM_LND_MACAROON").unwrap();
+            let lnd_tls_cert = env::var("FM_LND_TLS_CERT").unwrap();
+            let lnd_client = connect(
+                lnd_rpc_addr.clone(),
+                lnd_tls_cert.clone(),
+                lnd_macaroon.clone(),
+            )
+            .await
+            .unwrap();
+            let rpc_lnd = Arc::new(Mutex::new(lnd_client.clone()));
+            let lightning = RealLightningTest::new(rpc_cln, rpc_lnd, gateway_node.clone()).await;
```

Comment:
Do we construct both the CLN rpc client and the LND rpc client because both might be used in the test, but only one is used for the gateway?

Reply:
make a comment about why 2 are required.

Reply:
done